extensions/wikia/CreatePage/js/CreatePage.js

```diff
@@ -288,6 +299,18 @@ var CreatePage = {
 	init: function( context ) {
 		'use strict';
 		CreatePage.context = context;
+
+		$( '#contribute-button-create-page' ).click(function() {
+			CreatePage.trackCreatePageStart(CreatePage.CREATE_PAGE_CONTRIBUTE_BUTTON);
+			CreatePage.flowName = CreatePage.CREATE_PAGE_CONTRIBUTE_BUTTON;
+		});
+
+		$( function() {
+			$( '#WikiaArticle').find( 'a.new' ).toArray().forEach(function(redlink) {
+				redlink.href = redlink.href + '&flow=' + CreatePage.CREATE_PAGE_REDLINK;
```

Comment:
Could you use `querystring` module here in case the url will change in time? So we can support `#`.